doc/default.nix

```
@@ -20,7 +20,33 @@ in pkgs.stdenv.mkDerivation {
     ln -s ${doc-support} ./doc-support/result
   '';
 
+  epub = ''
+    <book xmlns="http://docbook.org/ns/docbook"
+          xmlns:xlink="http://www.w3.org/1999/xlink"
+          version="5.0"
+          xml:id="nixpkgs-manual">
+      <info>
+        <title>Nixpkgs Manual</title>
+        <subtitle>Version ${pkgs.lib.version}</subtitle>
+      </info>
+      <chapter>
+        <title>Temporarily unavailable</title>
+        <para>
+          The Nixpkgs manual is currently not available in ePub format,
+          please use the <link xlink:href="https://nixos.org/nixpkgs/manual">HTML manual</link>
+          instead.
+        </para>
+        <para>
+          If you've used the ePub manual in the past and it has been useful to you, please
```

```suggestion
          The Nixpkgs manual is currently not available in EPUB format,
          please use the <link xlink:href="https://nixos.org/nixpkgs/manual">HTML manual</link>
          instead.
        </para>
        <para>
          If you've used the EPUB manual in the past and it has been useful to you, please
```

Apparently this is official spelling?